lib/tic_tac_toe/position_validator.ex

```diff
@@ -1,12 +1,14 @@
 defmodule TicTacToe.PositionValidator do
-  alias TicTacToe.Board
+  alias TicTacToe.BoardState
 
   def error(position, board) do
+    board = BoardState.available_positions(board)
```

Comment:
Does `available_positions` update the board? If so, we could push that down into the function that modifies that state. Also, I think we could probably do this with `BoardInspect.get`